templates/master/00-master/_base/files/usr-local-bin-etcd-snapshot-restore-sh.yaml

```diff
@@ -9,23 +9,34 @@ contents:
     set -o pipefail
 
     # example
-    # ./etcd-snapshot-restore.sh $path-to-snapshot $inital_cluster
+    # ./etcd-snapshot-restore.sh $path-to-backup-dir $inital_cluster
 
     if [[ $EUID -ne 0 ]]; then
       echo "This script must be run as root"
       exit 1
     fi
 
     usage () {
-        echo 'Path to snapshot and initial cluster are required: ./etcd-snapshot-restore.sh $path-to-snapshot $initial_cluster'
+        echo 'Path to backup dir and initial cluster are required: ./etcd-snapshot-restore.sh <path-to-backup-dir> $INITIAL_CLUSTER'
         exit 1
     }
 
     if [ "$1" == "" ] || [ "$2" == "" ]; then
         usage
     fi
 
-    SNAPSHOT_FILE="$1"
+    if [ ! -d $1 ]; then
+      echo "$1 is not a directory. Please specify backup-dir"
```

Comment:
if this is getting backported we need to consider any backups that existed which did not have these additional assets

Reply:
I agree. I can accommodate both file and directory and only restore the snapshot and not delete the static pod directories. Thanks.

Reply:
we at least need to consider what would happen and possibly document workaround.